### ISSUE TITLE
Power pellet time duration

### ISSUE THREAD

**Issue by JeremyPlott** [DesrialBecker/PAC-MAN-With-Friends#17]
It is currently infinite. Needs to last for x amount of time, and then switch ghost state back to Ghost.GhostState.Chase  once the duration has ended.

**Comment by CoreyMeyerCin** [comment]
I tried to fix this issue yesterday, and I was unable to find out a good way to do it. Invoke() only works on methods that take 0 parameters but Invoke must be used on static methods. Thoughts? Coroutine had a lot of issues that I did not have time to look at. Thoughts on using either of those?

I know there is also a Wait function(dont remember the syntax) but I do not recall how it actually functions.
